libbpfgo/libbpfgo.go

```diff
@@ -688,6 +703,7 @@ func (rb *RingBuffer) Close() {
 	}
 	rb.Stop()
 	C.ring_buffer__free(rb.rb)
+	eventChannels.Remove(rb.slot)
```

Comment:
Didn't `rb.Stop()` (two lines above) take care of this already?

Reply:
Great catch! It does but it needs to be done here since it was inserted in Init. Otherwise someone can call Stop() and then Start() and it'll break.

Reply:
I've removed the one in Stop